test/extended/machines/workers.go

```diff
@@ -0,0 +1,211 @@
+package operators
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/stretchr/objx"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/dynamic"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	// TODO: cloud team may change this when changing label keys
+	machineLabelSelectorWorker = "sigs.k8s.io/cluster-api-machine-role=worker"
```

Comment:
@spangenberg This will never pass the test with that label unless https://github.com/openshift/installer/pull/1263 gets merged, which is waiting to be rebased